sdk/python/kfp/gcp.py

```diff
@@ -121,3 +121,16 @@ def _set_preemptible(task):
     return task
 
   return _set_preemptible
+
+def use_gpu_nodepool(toleration: V1Toleration = V1Toleration(
```

Comment:
So if I understand it correctly this should be called `add_gpu_toleration`, right?

Reply:
Ah yes. I'll fix it. Thanks!